AEXS/aex-141.md

```diff
@@ -28,32 +28,30 @@ The following standard describes standard interfaces for non-fungible tokens. Th
 ```sophia
 contract interface NFT =
     datatype metadata_type = URL | IPFS | OBJECT_ID | MAP
-    datatype metadata = string | map(string, string)
+    datatype metadata = String(string) | Map(string, string)
```

Comment:
@arjanvaneersel also here I am not sure if that's needed. I think we can only use `String` in the interface. any comment on that @thepiwo @bogdan-manole @ghallak ?

Reply:
With me it wouldn't compile without the String. It's because of the variatype String takes a string.

Reply:
I will let some Sophia dev have another look at the definition and the example implementation. but I think we have to keep it like you defined it 👍